src/test/java/org/generationcp/middleware/data/initializer/InventoryDetailsTestDataInitializer.java

```diff
@@ -18,22 +13,36 @@
 import org.generationcp.middleware.pojos.ims.Transaction;
 import org.generationcp.middleware.pojos.report.TransactionReportRow;
 
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
 public class InventoryDetailsTestDataInitializer {
 
 	private static final int PERSON_ID = 1;
 	private static final String LIST_SOURCE_TYPE = "LIST";
 	private static final String GERMPLASM_ENTITY_TYPE = "GERMPLSM";
 	private static final int USER_ID = 1;
 	private static final int NO_OF_STOCK_LIST_ENTRIES = 20;
-
-	public static final Integer DATE = 19122016;
 	public static final String LIST_NAME = "List1";
 	public static final String USER = "User";
 	public static final String STATUS = "Active";
 	public static final Double AMOUNT = -50.0;
+	public Date DATE;
```

Comment:
@vmaletta if it is not a constant, it should not be uppercase

Reply:
Got it!